plugins/catalog-backend-module-incremental-ingestion/src/database/IncrementalIngestionDatabaseManager.ts

```diff
@@ -285,40 +298,41 @@ export class IncrementalIngestionDatabaseManager {
    * @param ingestionId - string
    * @returns All entities to remove for this burst.
    */
-  async computeRemoved(provider: string, ingestionId: string) {
+  async computeRemoved(provider: string, providerId: string) {
```

Comment:
Was this an unintentional rename? Something's up here

Reply:
Yep, that was a goof. I'll fix it.